### PR TITLE
Tweak CSS (visual)

### PR DESCRIPTION
**Pull request by carolynyoo**
Because the red bothered me...
- Navbar color blue scheme
- Margins on product page
- Typeface

screen shot 2015-04-26 at 2 33 54 pm

### CODE REVIEW

File: browser/scss/main.scss [hunk 1]
```diff
@@ -15,14 +16,4 @@
...
-.movie-card {
-    width: 160px;
-    height: 240px;
-    float: left;
-}
-
-.film-title {
-    font-weight: bold;
-}
+@import 'films/main';
```
Comment (itsaliu90) [inline browser/scss/main.scss]: modular-rific :fallen_leaf:

**Comment by itsaliu90** [comment]
:hamburger:

**Comment by darksim735** [comment]
I approve - looks like there's a merge conflict though. Feel free to handle locally and push.

**Comment by carolynyoo** [comment]
cool merging!

**Pull request merged**